packages/loot-core/vitest.config.ts

```diff
@@ -26,9 +26,12 @@ export default defineConfig({
     },
   },
   resolve: {
-    alias: {
-      '@actual-app/crdt': path.resolve('../crdt'),
-    },
+    alias: [
+      {
+        find: /^@actual-app\/crdt(\/.*)?$/,
+        replacement: path.resolve('../crdt/src$1'),
```

Comment:
This is kind of ugly, but I'm not sure there's a better way. Essentially, the `crdt` package is built into a `dist` directory and the `package.json` defines all files in there as the distribution. We need some way to get `vite` to prefer local files when running tests, and the easiest way to do that is just to point to a subdirectory